src/modal.tsx

```diff
@@ -658,7 +759,7 @@ class ReactNativeModal extends Component {
 
         {avoidKeyboard && (
           <KeyboardAvoidingView
-            behavior={Platform.OS === 'ios' ? 'padding' : null}
+            behavior={Platform.OS === 'ios' ? 'padding' : undefined}
```

Comment:
Just a comment: I *think* that `null` had actually a different behaviour than `undefined` waaay back when we added the feature but looking at the docs it seems that using `undefined` is safe now so 👍

Reply:
Just checked, seems it's tested into a switch so undefined and null both works the same way, but it's typed against undefined so I did this to allow TS to pass